docusaurus.config.js

```diff
@@ -4,54 +4,71 @@
 // There are various equivalent ways to declare your Docusaurus config.
 // See: https://docusaurus.io/docs/api/docusaurus-config
 
-import { themes as prismThemes } from "prism-react-renderer";
+import { themes as prismThemes } from 'prism-react-renderer';
+import fauxRemarkEmbedder from '@remark-embedder/core';
+import fauxOembedTransformer from '@remark-embedder/transformer-oembed';
+
+const transformer = {
+  ...fauxOembedTransformer,
+  getHTML: async (url, getConfig = {}) => {
+    let html = await fauxOembedTransformer.getHTML(url, getConfig);
+    if (html?.startsWith('<iframe')) {
+      html = html.replace('width="200"', 'class="iframe-video"');
+      html = html.replace('height="113"', '');
+    }
```

Comment:
for what do we need it?

Reply:
we have added the support for rendering of videos